src/components/profile/profile.component.js

```diff
@@ -69,6 +85,14 @@ class Profile extends React.Component {
 Profile.propTypes = {
   /** [Legacy] A custom class name for the component */
   className: PropTypes.string,
+  /* Custom source URL */
+  src: ({ src }) => {
```

Comment:
What's the purpose of having this validation function? Couldn't we use a simple `string` check instead?
```js
src: PropTypes.string
```